Mathlib/RingTheory/AlgebraicIndependent.lean

```diff
@@ -82,6 +83,21 @@ theorem algebraicIndependent_empty_type_iff [IsEmpty ι] :
     AlgebraicIndependent R x ↔ Injective (algebraMap R A) := by
   rw [algebraicIndependent_iff_injective_aeval, MvPolynomial.aeval_injective_iff_of_isEmpty]
 
+/-- An element `x` is transcendental if and only if the one-element family `![x]` is
+algebraically independent. -/
+theorem transcendental_iff_algebraicIndependent {x : A} :
```

Comment:
I think it's useful to have a `AlgebraicIndependent R x ↔ Transcendental R (x default)` given `Unique ι` as a simp lemma.

Reply:
Good idea. Will be added soon.